pkg/kubelet/dockershim/docker_image.go

```diff
@@ -135,13 +135,20 @@ func (ds *dockerService) RemoveImage(_ context.Context, r *runtimeapi.RemoveImag
 	// An image can have different numbers of RepoTags and RepoDigests.
 	// Iterating over both of them plus the image ID ensures the image really got removed.
 	// It also prevents images from being deleted, which actually are deletable using this approach.
```

Comment:
What does this last sentence in the comment refer to?

Reply:
I think "it" -> "An image [having] different numbers of RepoTags and RepoDigests".  The comment could definitely use clarification.

Reply:
or removal

Reply:
fixed